### DISCUSSION TITLE
Consider using `CGO_ENABLED=1` for default macOS build

### DISCUSSION THREAD

**Discussion by karniv00l** [pocketbase/pocketbase#871] [Q&A] [answered]
I've personally encountered the same issue for every Go app for mac that makes external requests (docker compose, flyctl, Pocketbase). 

Issue is that whenever request happens, in case of Pocketbase it's OAuth request, it hangs my network stack and system reboot is required.

For details, please see: 

- https://github.com/docker/compose/issues/9930
- https://github.com/planetscale/discussion/discussions/181

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, that's very strange behavior. Thanks for letting me know.

I don't want to set `CGO_ENABLED=1` because the binary will no longer be statically linked and I don't have access to a mac to test it whether it will work if it is built with the goreleaser action.

Let's wait a little to see what will be the resolution of https://github.com/golang/go/issues/52839 (recently there was also an announcement for "PRIVATE" security go fix that will be available on November 1, so it may be related - https://groups.google.com/g/golang-announce/c/dRtDK7WS78g?).

**Reply by ganigeorgiev** [reply]
_Just to add some recent info if anyone stumble on this - there seems to be already a patch for an issue that looks very similar to the described one in https://go-review.googlesource.com/c/go/+/448020?tab=comments. It is already merged, so most likely it will be available with the next Go 1.20._
